charts/k8s-infra/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: k8s-infra
 description: Helm chart for collecting metrics and logs in K8s
 type: application
-version: 0.11.10
+version: 0.11.20
```

Comment:
shouldn't this be `0.11.11`? :eyes:

Reply:
There is only single change with kubeletstats metrics preset

Reply:
@coderabbitai please bump the version to 0.11.11 instead of 0.11.20